nwapp/utils/notifier.js

```diff
@@ -8,11 +22,9 @@ var win = gui.Window.get();
 var CLIENT = require('./client-type');
 var SOUNDS_ITEMS = require('../components/sound-items');
 
-// TODO: Resize images because GitHub is not resizing default avatars and it creates inconsistent avatar size
-// TODO: Deal with other platforms
-function getNotificationImageForUri(uri) {
-  if (!uri) return "";
-  if (CLIENT.match(/^osx|^win/)) return "https://avatars.githubusercontent.com/" + uri.split("/")[1] + "?s=32";
+function getFullUrlNotificationImageForUri(uri) {
```

Comment:
Use the `icon` property from the original faye message. It's a little more future-proof.